components/fallen_defenders.py

```diff
@@ -55,7 +54,7 @@ def style_df(df):
 
     fallen_df = pd.DataFrame(fallen).sort_values(["Best #", "days", "Best wave"]).reset_index(drop=True)
 
-    fallen_champ = fallen_df[fallen_df["Best #"] == 1]
+    fallen_legend = fallen_df[fallen_df["Best #"] == 1]
```

Comment:
Fallen defenders I'm thinking will just stay champ forever, but `today` will be fixed do some date in October 2024. Otherwise, it has to be made working with _both_ champ and legends, otherwise it completely looses it's meaning.

But for now I'm leaning towards sealing this and not touching it anymore unless I have time to make it work with both leagues.